File: 001-099/02/2.cpp
#include <iostream>

int main() {
    int a = 0, b = 1;
    int total = 0;
    int tmp;
    while (b < 4000000) {
        tmp = a;
        a = b;
        b += tmp;
        if (b % 2 == 0) {
            total += b;
        }
    }
    std::cout << total << std::endl;
}

File: 001-099/02/2.php
<?php
$a = 0;
$b = 1;
$total = 0;
while ($b < 4000000) {
    if ($b % 2 == 0) {
        $total += $b;
    }
    $tmp = $a;
    $a = $b;
    $b += $tmp;
}
print "$total\n";
?>
